src/utils/CustomDeleteDialog.jsx

```diff
@@ -0,0 +1,71 @@
+import { useA11yDialog } from 'react-a11y-dialog';
```

Comment:
At first when I started to read through this code and saw useA11yDialog I was like what the heck is this naming convention!? Then I realized its a library I've never seen before, oops [😂](https://emojipedia.org/face-with-tears-of-joy/)
Really cool though, I like the modal! and the docs online seemed pretty straightforward!